sale_force_invoiced/__manifest__.py

```diff
@@ -4,12 +4,15 @@
 {
     "name": "Sale Force Invoiced",
     "summary": "Allows to force the invoice status of the sales order to Invoiced",
-    "version": "17.0.1.0.0",
+    "version": "17.0.1.1.0",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
     "category": "sale",
     "license": "AGPL-3",
     "website": "https://github.com/OCA/sale-workflow",
-    "depends": ["sale"],
-    "data": ["view/sale_order.xml"],
+    "depends": ["sale", "sales_team"],
```

Comment:
No need to include sales_team, as it's already a dependency of sale

Reply:
Done :+1: